broker/src/main/java/io/zeebe/broker/clustering/topology/TopologyManagerImpl.java

```diff
@@ -189,9 +189,22 @@ private BrokerInfo readBrokerInfo(final Member eventSource) {
     final BrokerInfo brokerInfo = BrokerInfo.fromProperties(eventSource.properties());
     if (brokerInfo != null && !isStaticConfigValid(brokerInfo)) {
       LOG.error(
-          "Static configuration of node {} differs from local node {}",
+          "Static configuration of node {} differs from local node {}.",
           eventSource.id(),
           atomix.getMembershipService().getLocalMember().id());
+
+          LOG.error(
+            "Static Configuration :: Local Node Configuration \n" 
+             + "NodeId: {} >= 0 \n"
```

Comment:
Please change this such that it states that node id should be >= 0 and < localBroker.getClusterSize()